models/index.js

```diff
@@ -23,11 +23,11 @@ const logger = require("./../utils/logger");
 const sequelize = new Sequelize(DB_NAME, DB_USER, DB_PASSWORD, {
   host: DB_HOST,
   dialect: DB_DIALECT,
-  dialectOptions: {
-    ssl: {
-      SSL
-    }
-  }
+  // dialectOptions: {
```

Comment:
Chief no forget

Reply:
Lol. Just uncomment it. I simply commented it because I used my local db.